src/src/PluginStructs/P104_data_struct.h

```diff
@@ -415,11 +415,11 @@ struct P104_data_struct : public PluginTaskData_base {
   int8_t                   cs_pin;
   uint8_t                  modules = 1u;
 
-  bool    initialized   = false;
-  int8_t  expectedZones = -1;
-  int8_t  previousZones = -1;
-  uint8_t numDevices    = 0;
-  uint8_t zoneOrder     = 0;
+  bool     initialized   = false;
+  int8_t   expectedZones = -1;
+  int8_t   previousZones = -1;
+  uint16_t numDevices    = 0;
+  uint8_t  zoneOrder     = 0;
 
   String error;
```

Comment:
Just as a generic tip, since it will not make a lot of difference here...
Orrdering the members from biggest to smallest may result in less memory usage.
`String` > `uint16_t` > `int8_t` > `bool`

Reply:
I've reorganized it a bit.